### PR TITLE
Objects search

### PR DESCRIPTION
**Pull request by danil-topchiy**
Based on #5

### CODE REVIEW

File: src/edap/edap.py [hunk 1]
```diff
@@ -54,6 +68,27 @@ def add_user(self, uid, name, surname, password):
...
+
+    def get_user(self, uid):
+        """
+        Search in subobjects of organizational unit "people" by uid
+        Args:
+            uid (str):
+
+        Returns:
+        """
+        return self.get_users(search="uid={}".format(uid))
```
Comment (matejak) [inline src/edap/edap.py]: It is probably safer to use `f"uid={uid}"`

**Review by matejak — CHANGES_REQUESTED** [review]
This change is good and useful, there are just minor string issues.
Could you rebase against the current master and also add unit tests for newly added functionality?

**Comment by danil-topchiy** [comment]
Rebased, added tests and changed to f-strings.

**Review by matejak — APPROVED** [review]
Looks great, thanks!

**Pull request merged**